app/assets/stylesheets/components/_modals.scss

```diff
@@ -1,75 +1,85 @@
-.modal {
-    font-size:13px;
-    padding:20px 30px;
-    width:340px;
-
-    h2 {
-        margin:0 0 2px;
-    }
+.modal-overlay {
+    background: rgba(#000, 0.85);
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 20;
 
-    .subheader {
-        color:#888;
-        font-size:14px;
-        margin:0 0 15px;
+    &.show {
+        display: block;
     }
+}
 
-    fieldset {
-        border-top:1px solid #ddd;
-        margin:0 -30px;
-        padding:15px 30px;
-    }
+.modal {
+    background: #fff;
+    margin: 0 auto;
+    padding: 40px;
+    position: absolute;
+        top: 50%;
+        left: 50%;
+    width: 500px;
+    -webkit-transform: translate(-50%, -50%);
+    -moz-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    -webkit-border-radius: 6px;
+    -moz-border-radius: 6px;
+    border-radius: 6px;
 
-    label {
-        clear:both;
-        display:inline-block;
-        float:left;
-        font-size:14px;
-        font-weight:700;
-        line-height:34px;
-        width:100px;
+    * + * {
+        margin-top: 1.5em;
     }
 
-    .field {
-        display:inline-block;
-        float:left;
-        margin:0 0 15px;
+    .margins-off {
+        > * + * {
+            margin-top: 0;
+        }
     }
+}
 
-    input[type="text"] {
-        border:1px solid #ccc;
-        font-size:13px;
-        line-height:20px;
-        padding:4px;
-        border-radius:3px;
-        -moz-border-radius:3px;
-        -webkit-border-radius:3px;
-    }
+.modal--alert {
+    text-align: center;
+}
 
-    .actions {
-        margin-left:100px;
+.modal__header {
+    .close {
+        font-size: 18px;
+        position: absolute;
+        top: 10px;
+        right: 10px;
 
-        a {
-            margin-left:15px;
+        &:hover {
+            color: $color-gray-light;
         }
     }
+}
 
-    .button {
-        background: #f5f5f5;
-        border: 1px solid #aaa;
-        color:#000;
-        font-size:11px;
-        padding:3px 6px 3px;
+.modal__title,
```

Comment:
Selector `modal__title` should be written in lowercase with hyphens<br>Selector `modal__subtitle` should be written in lowercase with hyphens